var/spack/repos/builtin/packages/py-reportlab/package.py

```diff
@@ -19,3 +19,5 @@ class PyReportlab(PythonPackage):
     # - easy_install, provided by py-setuptools
     # - pip, provided by py-pip
     extends('python', ignore=r'bin/.*')
+
+    depends_on('freetype')
```

Comment:
There's also a bunch of other missing deps:
```python
install_requires=['pillow>=2.4.0','pip>=1.4.1', 'setuptools>=2.2'],
```
Also, it looks like there are a few env vars you can set to help the build find freetype if you notice any linking issues.